src/site/content/en/blog/tools-for-extracting-critical-css/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Tools for extracting critical CSS
+subhead: Learn how to improve render times with critical CSS technique.
+authors:
+  - mihajlija
+date: 2019-05-07
+hero: hero.jpg
+alt: A description of the hero image for screen reader users.
+description: |
+  Learn how to improve render times with critical CSS technique and how to choose the best tool for your project.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+  - css
+---
+
+The browser must download and parse CSS files before it can show the page, which makes CSS a render-blocking resource. If CSS files are big, or network conditions are poor, requests for CSS files can significantly increase the time it takes for a web page to render.
+
+{% Aside 'key-term' %}
+Critical CSS is a technique that extracts the CSS for above-the-fold content in order to get a meaningful render as fast as possible.
+{% endAside %}
+
+<figure class="w-figure  w-figure--center">
+  <img class="" src="./above-the-fold.png" alt="An illustration of a laptop and a mobile device with web pages overflowing the edges of screens" style="max-width: 600px;">
+</figure>
+
+{% Aside 'note' %}
+Above-the-fold is all the content a viewer sees on page load, before scrolling. There is no universally defined pixel height of what is considered above the fold content since there is a myriad of devices and screen sizes.
+{% endAside %}
+
+Inlining extracted styles in the `<head>` of the HTML document eliminates the need to make an additional request to fetch these styles. The remainder of the CSS can be loaded asynchronously.
```

Comment:
Reading the current text, i think we could tweak our description of how roundtrips to first render should be minimized. We're saying here that you should keep your HTML and your inline CSS under 14KB. Some of our prior guidance has said it's the above-the-old content that should fit under 14KB.

"…the server can send up to 10 TCP packets on a new connection (~14KB) in first roundtrip, and then it must wait for the client to acknowledge this data before it can grow its congestion window and proceed to deliver more data.

Due to this TCP behavior, it is important to optimize your content to minimize the number of roundtrips required to deliver the necessary data to perform the first render of the page. Ideally, the ATF (above the fold) content should fit under 14KB — this allows the browser to paint the page after just one roundtrip…"

https://developers.google.com/speed/docs/insights/mobile#delivering-the-sub-one-second-rendering-experience